crates/solidity/outputs/cargo/crate/src/backend/binder/scopes.rs

```diff
@@ -119,7 +118,7 @@ impl Scope {
             Self::File(file_scope) => file_scope.node_id,
             Self::Function(function_scope) => function_scope.node_id,
             Self::Modifier(modifier_scope) => modifier_scope.node_id,
-            Self::Parameters(parameters_scope) => parameters_scope.node_id,
+            Self::Parameters(_) => unreachable!("parameters scope don't have a node ID"),
```

Comment:
I wonder why this change?

Reply:
Because of the simplification, we no longer have a `NodeId` for the `ParametersDeclaration`, which was used to create this scope, and the parameters are direct children of the `FunctionDefinition`. We cannot the definition node ID because that conflicts with the function scope itself. But it doesn't really matter and we don't need the parameters scope to be indexable by node as it's always related to a function. In fact, the only purpose for declaring them separately (as opposed to in the function scope itself) is to resolve arguments in named argument call expressions.